test/system/helpers.bash

```diff
@@ -1,5 +1,11 @@
 # -*- bash -*-
 
+# Ignore cgroups-v1 warning in cirrus
```

Comment:
What it does, will be obvious to a maintainer of this.  If not here maybe in a followup PR, would you consider commenting WHY this is being set to ignore only for cirrus-ci?

I hate to ask for a re-spin of CI just for a comment update.

Reply:
(pls double-check my logic there, my :thinking: :tophat: isn't working well this morning)

Reply:
hmm, it should be set on all CI. If the user wants to run the tests manually on their local cgroupsv1 system, I think they should set this env var manually as well. I think it should be `"$CI" == "true"` without the preceeding negation then.

@cevich unless the test can be set to accept this warning. Would that work?

Reply:
I wonder if the debian image should have cgroups v2 instead of v1. The default debian 12 image on GCE is v2.

Reply:
I would assume we want some cgroupv1 testing for RHEL?

Anyhow would it not be easier to set the env var in cirrus so it will not be set for downstream gating CI. If I read this correctly the goal is to patch the warning out for RHEL 9 so the gating tests there should obviously NOT have the env set so we know the warning removal patch actually works and is not silenced vie env var.

Reply:
I think switching debian to cgroups v2 would be the right way forward since we're deprecating it. Any cgroups v1 testing is better off being rhel9 / c9s only.

Reply:
yes that is a fair point but at the same time catching bugs only in RHEL gating is painful for everyone involved.